### PR TITLE
Fall 2020 Application Updates

### PR DESCRIPTION
**Pull request by aivantg**
Updated the website for fall 2020 application questions

### CODE REVIEW

File: app/views/student_applications/new.html.slim [hunk 1]
```diff
@@ -13,7 +13,7 @@
   p.static-page-desc.marginBot-md
     = t '.banner'
     | The application is due on
-    strong.apply-page-due< Friday, January 31st at 11:59 pm.
+    strong.apply-page-due< Friday, September 4th at 11:59 pm.
```
Comment (wangannie) [inline app/views/student_applications/new.html.slim]: Add time zone

File: db/seeds/questions.yml [hunk 2]
```diff
@@ -52,17 +57,23 @@ commitments:
...
+  question_category: more_questions
   question_type: checkbox
 
 # Designer/Dev application type
 
+fa20_application_type:
+  title: What positions are you applying to this semester?
+  hint:
+    'You may apply to more than one, but if you get accepted for more than one, you must make a decision for which
+    application to continue by September 6 at 11:59 PM. You only need to fill out one application regardless of how many positions you apply for.'
```
Comment (wangannie) [inline db/seeds/questions.yml]: Add timezone

File: db/seeds/questions.yml [hunk 3]
```diff
@@ -76,27 +87,27 @@ application_type:
...
   required: true
 
 available_for_meet_the_club:
   title: Which Meet the Club section are you able to attend?
   hint: Meet the Club is an event in our application process that you may be invited to after application decisions are released. This gives you and us an opportunity to get to know each other. You only need to attend one section.
-  question_category: general
+  question_category: more_questions
   question_type: multiple_choice_radio
-  choices: February 5th 8-10 PM,February 6th 8-10 PM,Both,Neither
+  choices: September 9th 6:30-8 PM,September 10th 6:30-8 PM,Both,Neither
```
Comment (wangannie) [inline db/seeds/questions.yml]: Add time zones

File: db/seeds/questions.yml [hunk 4]
```diff
@@ -76,27 +87,27 @@ application_type:
   choices: Developer,Designer,Developer and designer
 
 available_for_bp_games:
-  title: Will you be able to attend BP Games?
-  hint: BP Games is an event in our application process that you may be invited to after application decisions are released. BP Games will be on February 8, from 3-5 PM.
-  question_category: general
+  title: Are you be able to attend at least one of the BP Games sessions?
```
Comment (wangannie) [inline db/seeds/questions.yml:90]: I'd suggest changing all instances to Blueprint Games?

File: db/seeds/questions.yml [hunk 5]
```diff
@@ -76,27 +87,27 @@ application_type:
...
+  hint: 'BP Games is an event in our application process that you may be invited to after application decisions are released. There will be 3 BP Games sessions: Sept 11 6-8PM, Sept 12 10-12PM, Sept 12 12-2PM'
```
Comment (wangannie) [inline db/seeds/questions.yml]: This should only show for Developer applicants

File: db/seeds/questions.yml [hunk 6]
```diff
@@ -109,6 +120,27 @@ design_portfolio:
...
 
+fa20_mission_first:
+  title: 'What does social good mean to you?'
+  hint: "If you're applying for a Developer/Designer position, please include: Why do you want to build/design applications in the social impact space?<br/><br/>If you're applying for an External position, please include: Why do you want to promote tech for social good?<br/><br/>Do not use this space to talk about Blueprint — we want to hear about you! Please limit your response to 200 words."
+  question_category: short_answer
+  question_type: long_text
+  word_limit: 200
+
+fa20_perpetual_growth:
+  title: Talk about a time where you went through a period of growth. What motivated you, what challenged you, and how has it impacted you today?
```
Comment (wangannie) [inline db/seeds/questions.yml:131]: nit: change 'where' to 'when'

File: db/seeds/questions.yml [hunk 7]
```diff
@@ -109,6 +120,27 @@ design_portfolio:
 
 # Short answer questions
 
+fa20_mission_first:
+  title: 'What does social good mean to you?'
+  hint: "If you're applying for a Developer/Designer position, please include: Why do you want to build/design applications in the social impact space?<br/><br/>If you're applying for an External position, please include: Why do you want to promote tech for social good?<br/><br/>Do not use this space to talk about Blueprint — we want to hear about you! Please limit your response to 200 words."
```
Comment (wangannie) [inline db/seeds/questions.yml:125]: Change Developer/Designer to Developer or Designer.

Not a huge deal but I could see this being misinterpreted as roles can be blended

File: db/seeds/questions.yml [hunk 8]
```diff
@@ -130,7 +162,41 @@ be_humble:
...
+
+fa20_technical_gauge:
+  title: '[Optional] If you are applying as a developer, from the following courses, list out the ones you have taken/are currently taking:'
+  hint: "- CS 10\n- CS 61A\n- Data 8\n- CS 88\n- CS 61B\n- Atleast 1 CS upper div course (answer yes or no)\n- Other: (could include relevant decals such as the Web Design/React/Rails Decals or other learning experiences)"
+  required: false
+  question_type: short_text
+  question_category: background
+
+fa20_classes_additional_info:
+  title: '[Optional] Feel free to add anything else we should know about your answer to the above (ex. currently auditing, applied for WDD, took a class but dropped partway due to COVID).'
```
Comment (wangannie) [inline db/seeds/questions.yml:175]: nit: add an option for 'couldn't get in from waitlist' bc that was me first semester ;__;

File: db/seeds/questions.yml [hunk 9]
```diff
@@ -130,7 +162,41 @@ be_humble:
...
+  hint: There is no need to elaborate on the classes you have taken beyond the ones on the list above (for example, we don't need to specific upper div courses). Please limit your response to 100 words.
+  required: false
+  question_category: background
+  question_type: long_text
+  word_limit: 100
+
+fa20_background_question:
+  title: '[Optional] Is there any additional information or circumstances we should know about to understand you better? This can be anything related to your identity, current/previous access to relevant resources, a personal situation due to COVID, etc. In reading your application, we want to ensure that we understand and respect your background, experiences, and perspectives.'
```
Comment (wangannie) [inline db/seeds/questions.yml:183]: Change to "Are there any additional circumstances you’d like to us to know about to better understand you?" 
just for grammar

File: db/seeds/questions.yml [hunk 10]
```diff
@@ -174,6 +240,12 @@ why_no_retreat:
   question_type: long_text
   required: false
 
+fa20_heard_from:
+  title: How did you first hear about Blueprint?
+  hint: Please let us know how you first heard about Blueprint, whether it’s through social media, through a friend, through classes, through a certain Sproul-alternative, etc.
```
Comment (wangannie) [inline db/seeds/questions.yml:245]: I'd cut the first 'Please let us know how you first heard about Blueprint' bc it's rather redundant.

File: db/seeds/questions.yml (hunk 8 above)
Comment (aivantg) [reply db/seeds/questions.yml:176]: I think that's cloooose enough to just the "applying" part that it should be covered? It's awkward to phrase so can include it if you have suggestion for phrasing!

File: db/seeds/questions.yml [hunk 11]
```diff
@@ -52,51 +57,57 @@ commitments:
...
     continuing with your developer application."
   question_category: general
   question_type: multiple_choice_radio
   choices: Developer,Designer,Developer and designer
 
 available_for_bp_games:
-  title: Will you be able to attend BP Games?
-  hint: BP Games is an event in our application process that you may be invited to after application decisions are released. BP Games will be on February 8, from 3-5 PM.
-  question_category: general
+  title: If you are a developer, are you be able to attend at least one of the Blueprint Games sessions?
```
Comment (wangannie) [inline db/seeds/questions.yml:90]: missed a space "at least"

File: db/seeds/questions.yml [hunk 12]
```diff
@@ -52,51 +57,57 @@ commitments:
...
+  hint: 'Blueprint Games is an event in our developer recruitment process that you may be invited to after application decisions are released.<br/><br/>There will be sessions: Sept 11 6-8PM PST, Sept 12 10-12PM PST, Sept 12 12-2PM PST.'
```
Comment (wangannie) [inline db/seeds/questions.yml:91]: add time zones

File: db/seeds/questions.yml [hunk 13]
```diff
@@ -52,51 +57,57 @@ commitments:
...
+  question_category: more_questions
   question_type: multiple_choice_radio
-  choices: Yes,No
+  choices: Yes,No,N/A
```
Comment (wangannie) [inline db/seeds/questions.yml:94]: can we make this something like N/A - not applying as a developer

File: db/seeds/questions.yml [hunk 14]
```diff
@@ -174,9 +241,15 @@ why_no_retreat:
...
+fa20_heard_from:
+  title: How did you first hear about Blueprint?
+  hint: Whether it’s through social media, through a friend, through classes, through a certain Sproul-alternative, etc.
+  question_category: more_questions
+  question_type: short_text
+
 heard_from:
   title: How did you first hear about Blueprint?
-  hint: Please let us know how you first heard about Blueprint, whether it’s through social media, through a friend, through classes, through Sproul, etc.
+  hint: Whether it’s through social media, through a friend, through classes, through Sproul, etc.
```
Comment (wangannie) [inline db/seeds/questions.yml:252]: Replace first 2 words with 'For example,'

I'd also add event or club fair instead of Sproul

**Review by wangannie — DISMISSED** [review]
Few more minor changes + typo fix but otherwise LGTM! Thank you for being so patient.

**Comment by aivantg** [comment]
Merging after slack feedback

**Pull request merged**